src/routes/docs/tutorials/nuxt/step-2/+page.markdoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: tutorial
+title: Build an ideas tracker with Nuxt
+description: Learn to build an idea tracker app with Appwrite and Nuxt
+step: 2
+---
+
+# Create Nuxt project {% #create-nuxt-project %}
+
+Create a Nuxt app with the `npx init` command.
+
+
+```sh
+npx nuxi@latest init ideas-tracker
+```
+
+# Add dependencies {% #add-dependencies %}
+
+Install the JavaScript Appwrite SDK.
+
+```sh
+npm install appwrite
+```
+
+You can start the development server to watch your app update in the browser as you make changes.
+
+```sh
+npm run dev
+```
+
+# Pages and layouts for routing {% #pages-and-layout-for-routing %}
+
+In Nuxt, directories help organize the codebase and minimize boilerplate. The purpose is
+making it easy to find and manage different aspect of your application.
+
+The files added to the `pages` directory will automatically become a route. Nuxt will look for a layout (can be more than one) to accompany a page
+in the `layouts` directory. When these files have been created we can edit the `app.vue` file to render our pages. Now we will have a working app to verify our changes in the development server throughout the tutorial.
+As an additional step, by adding [Appwrite's Pink Design system]('https://pink.appwrite.io/') we will also have a global layout system to help with the layout.
+
+
+### 1. Add a page
+
+Create file `src/pages/index.vue` and add the following code:
+
+```vue
+<template>
+  <div>
+    <h1>Hello, idea tracker!</h1>
+  </div>
+</template>
+```
+
+
+### 2. Add a default layout
+
+ a file `src/layouts/default.vue` and insert the following code:
```

Comment:
Please make sure you're writing complete sentences. Use a period instead of `:`

Reply:
Removed